config/default/kustomization.yaml

```diff
@@ -1,16 +1,16 @@
 # Adds namespace to all resources.
-namespace: kamaji-system
+namespace: kamaji-core-system
```

Comment:
This would be a breaking-change, what's the reason for changing the target namespace?

Reply:
Using a dedicated one just for the CAPI requirements is better. But I am also not a kustomize expert.. I need to think about it.

Reply:
It seems possible to allow duplicates with the option `allow_duplicates`, but I don't know how to set this.

Reply:
it works after adding allow_duplicates.
```
ubuntu@kamaji-tilt:~/test$ k get po -n kamaji-system
NAME                                              READY   STATUS    RESTARTS   AGE
capi-kamaji-controller-manager-565b46fb8d-lnmbc   1/1     Running   0          44m
kamaji-controller-manager-7595db499c-7x5pg        1/1     Running   0          37m
```

CAPI Tiltfile
```
        ...
if p.get("kustomize_config", True):
        yaml = read_file("./.tiltbuild/yaml/{}.provider.yaml".format(name))
        k8s_yaml(yaml, allow_duplicates = True)
        objs = decode_yaml_stream(yaml)
        ...
```

Reply:
With your help :)